css/style.css

```diff
@@ -16,4 +16,406 @@ body {
+  justify-content: center;
+  align-items: center;
+  background-size: cover
+}
+
+.hero h1 {
+  margin-bottom: 1rem;
+  font-size: 2.5rem;
+}
+
+.hero p {
+  font-size: 1.2rem;
+  margin-bottom: 2rem;
+}
+
+/* Section */
+
+.article {
+  margin-bottom: 1rem;
+  border: grey solid 1px;
+  padding: 10px;
+}
+
+
+.section__thumbnail {
+  object-fit: contain;
+  width: 5rem;
+  width: 100px;
+  height: 100px;
+}
+
+.boxes {
+  display: flex;
+  justify-content: center;
+  column-gap: 80px;
+}
+
+.section__content {
+  color: black;
+  text-align: center;
+}
+
+.decor_pic {
+  width: 1200px;
+  left: 439px;
+  top: 1758px;
+  background: url(/level-2/homepage-feature.png);
+  transform: matrix(-1, 0, 0, 1, 0, 0);
+  background-size: cover;
+}
+
+.get_today {
+  width: 264px;
+  height: 70px;
+  left: 888px;
+  top: 2045px;
+  background: #E0633A;
+  border-radius: 8px;
+
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+  color: #FFFFFF;
+}
+
+.sectionbg {
+  width: 100%;
+  height: 439px;
+  left: 599px;
+  top: 1758px;
+  background: rgba(224, 99, 58, 0.1);
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  display: flex;
+
+}
+
+.orangecolor {
+  color: #E0633A;
+}
+
+
+
+/* Store */
+.store_content {
+  display: flex;
+  justify-content: flex-end;
+  column-gap: 250px;
+}
+
+
+.decor_photo {
+  width: 701px;
+  height: 1235px;
+  left: 739px;
+  top: 98px;
+}
+
+label {
+  width: 148px;
+  height: 25px;
+  left: 122px;
+  top: 414px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 21px;
+  line-height: 25px;
+  color: #666666;
+}
+
+select:invalid {
+  color: gray;
+  font-style: italic;
+}
+
+.order_button {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 22px;
+  line-height: 26px;
+  color: #FFFFFF;
+
+  width: 269px;
+  height: 70px;
+  left: 122px;
+  top: 1193px;
+  background: #E0633A;
+  border-radius: 4px;
+}
+
+.p3 {
+  width: 140px;
+  height: 25px;
+  left: 122px;
+  top: 940px;
+
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 21px;
+  line-height: 25px;
+  color: #666666;
+}
+
+.tc {
+  width: 466.04px;
+  height: 50px;
+  left: 156.96px;
+  top: 1107px;
+
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 140%;
+  color: #666666;
+}
+
+.tc_highlight {
+  color: #DE6E49;
+}
+
+.rect_1 {
+  width: 235px;
+  height: 53px;
+  left: 122px;
+  top: 451px;
+
+  border: 2px solid #666666;
+  border-radius: 4px;
+}
+
+.rect_2 {
+  width: 499px;
+  height: 53px;
+  left: 122px;
+  top: 573px;
+
+  border: 2px solid #666666;
+  border-radius: 4px;
+}
+
+.rect_3 {
+  width: 307px;
+  height: 53px;
+  left: 122px;
+  top: 817px;
+
+  border: 2px solid #666666;
+  border-radius: 4px;
+}
+
+
+.rect_4 {
```

Comment:
Doris nice job :)
just use a meaningful class name in your html so when you use them as a selector in css your code is more readable.